runtime/compiler/aarch64/codegen/ARM64JNILinkage.cpp

```diff
@@ -503,12 +502,11 @@ size_t J9::ARM64::JNILinkage::buildJNIArgs(TR::Node *callNode, TR::RegisterDepen
    if (numMemArgs > 0)
       {
       pushToMemory = new (trStackMemory()) TR::ARM64MemoryArgument[numMemArgs];
-
-      argMemReg = cg()->allocateRegister();
       }
 
    // align to 16-byte boundary
    totalSize = (totalSize + 15) & (~15);
+   TR_ASSERT_FATAL(totalSize <= 256, "totalSize must not be larger than 256");
```

Comment:
Please add a comment for the reason of this size limit.

Reply:
Added a comment.